eslint.config.mjs

```diff
@@ -0,0 +1,35 @@
+import globals from 'globals';
+import tsConfig from '@appium/eslint-config-appium-ts';
+
+export default [
+  ...tsConfig,
+  {
+    files: ['packages/support/**/*'],
+    languageOptions: {
+      globals: {
+        ...globals.node,
+        BigInt: 'readonly',
+      },
+    }
+  },
+  {
+    ...tsConfig.find(({name}) => name === 'Test Overrides'),
+    name: 'Test Support',
+    files: [
+      'packages/test-support/lib/**',
+      'packages/driver-test-support/lib/**',
+      'packages/plugin-test-support/lib/**',
+    ],
+  },
+  {
+    ignores: [
+      'examples/javascript-wd',
+      'sample-code',
```

Comment:
It looks like these paths no longer exist in the monorepo

Reply:
Looks like it's some legacy stuff from the Appium 1 era. Removed